### ISSUE TITLE
Prepocet cizi meny dle kurzu CNB

### ISSUE THREAD

**Issue by horsedriver** [pavelevap/ceske-sluzby#174]
V nekterem pripade shop prijima platby v cizi mene protoze chce, nebo musi. 

Plugin ale **neumi reportovat prepocet dle pozadavku EET**. Posila proste jen cislo ktere dostal. 

- Je treba dopsat tam aby v pripade cizi meny pred odeslanim dat do EET prepocital cizi menu dle aktualniho kurzu CNB

**Comment by pavelevap** [comment]
@horsedriver: Ano, není to zase tak časté a zatím to nikdo nechtěl :-) Ale neměl by být problém to dodělat, jak to máte nastavené? V eshopu je pouze jedna měna (v nastavení WooCommerce) nebo máte měn více (např. prostřednictvím nějakého pluginu)?

**Comment by horsedriver** [comment]
https://codecanyon.net/item/woocommerce-currency-switcher/8085217 - pouzivame tenhle plugin
Jako defaultni menu mame EUR, uz kvuli konverzi na paypalu, ze ktereho obcas platime zbozi z Ciny. Kdyz to jde rovnou do Kc, je to desna holba. 
S kolegou jsme nasadili ceske sluzby, s jednou malou upravou pluginu, ktery pouzivame na generovani pdf faktur do /my-account/ stranky woocommerce, ale k memu zdeseni jsem si blbe precetl ty nesmysly o EET a ti idioti si to neprepocitavaji u nich. Takze mam v evidenci platbu 3500EUR za objednavku reportovanou jako 3500KC.. :/

**Comment by pavelevap** [comment]
Tento plugin nepoužívám, takže s ním nemám zkušenosti a ani nevím, jak konkrétně funguje. Někdo tedy může na webu nakoupit v korunách, ale v objednávce máte pak interně všechno přepočtené na eura? Podle čeho si to přepočítává? Tam je patrně nějaký interní kurz euro/Kč? Neměl by se použít tento kurz, aby to pak všechno vycházelo? Budu se na to určitě muset nejdříve podrobněji podívat, [pošlete](https://www.separatista.net/kontakt) prosím přístupy (nebo někde zprovozněte testovací kopii webu).
